libraries/AP_OpticalFlow/AP_OpticalFlow_HereFlow.cpp

```diff
@@ -79,9 +79,11 @@ void AP_OpticalFlow_HereFlow::_push_state(void)
     float flowScaleFactorY = 1.0f + 0.001f * flowScaler.y;
     float integralToRate = 1.0f / integral_time;
     //Convert to Raw Flow measurement to Flow Rate measurement
-    state.flowRate = Vector2f(flowRate.x * flowScaleFactorX,
-                                flowRate.y * flowScaleFactorY) * integralToRate;
-    state.bodyRate = bodyRate * integralToRate;
+    state.flowRate = Vector2f{
+        rate_gyro_integral.x * flowScaleFactorX,
+        rate_gyro_integral.y * flowScaleFactorY
```

Comment:
```suggestion
        flow_integral.x * flowScaleFactorX,
        flow_integral.y * flowScaleFactorY
```

Reply:
I also prefer the suggested name change.  So flow_integral or flow_rate_integral would be good.

Reply:
Fixed, thanks.

I apparently mis-read the "no changes" output.  *Now* it's a no-compiler-output change